operator/cmd/loki-broker/main.go

```diff
@@ -22,7 +23,7 @@ type config struct {
 	Namespace string
 	Image     string
 
-	featureFlags  manifests.FeatureFlags
+	featureFlags  configv1.FeatureGates
```

Comment:
is there a reason we're keeping the featureFlags terminology here?

Reply:
Yes because here they are CLI flags.